packages/destination-actions/src/destinations/sendgrid/index.ts

```diff
@@ -0,0 +1,35 @@
+import type { DestinationDefinition } from '@segment/actions-core'
+import type { Settings } from './generated-types'
+
+import updateUserProfile from './updateUserProfile'
+
+const destination: DestinationDefinition<Settings> = {
+  name: 'Sendgrid Marketing Campaigns',
```

Comment:
Should the G be capitalized? Not sure what is correct so feel free to change my suggestions below to lower case the G instead.

```suggestion
  name: 'SendGrid Marketing Campaigns',
```